src/components/CommentsSection.jsx

```diff
@@ -1,38 +1,124 @@
 import axios from "axios";
 import { useEffect, useState } from "react";
+import { useContext } from "react";
+import { UserContext } from "../contexts/User";
 
-export default ({article_id}) => {
+export default ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [commentBody, setCommentBody] = useState("");
+  const [isEmpty, setIsEmpty] = useState(true);
+  const [disabled, setDisabled] = useState(false);
+  const { user, setUser } = useContext(UserContext);
 
-  const [comments, setComments] = useState([])
+  useEffect(() => {
+    axios
```

Comment:
consider extracting out your axios request in to a utils file to house them all together